services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -313,6 +325,7 @@ async def _remove_single_orphaned_service(
         )
         logger.info(message)
         try:
+            _log_stop_service(service_uuid, False)
```

Comment:
**two** levels of indirection just to display a log is unnecessary complexity!! 

You can create a message template if you do not want to repeat yourself and add all the prose in the comments that you like but it should suffice with a single 
```python
logger.warning( f"[HINT] Will stop service %s %s", f'{service_uuid=}', f'{save_state=}' )
```